packages/process/src/node/terminal-process.spec.ts

```diff
@@ -117,16 +117,16 @@ if (process.platform !== 'win32' || process.env.THEIA_PROCESS_TEST_OVERRIDE) {
 
         this.timeout(20_000);
 
-        interface ProcessExit extends IProcessExitEvent {
+        type ProcessExit = IProcessExitEvent & {
             output: string;
-        }
+        };
 
         // tslint:disable-next-line:no-any
         async function checkOutput(proc: TerminalProcess, pattern?: RegExp): Promise<ProcessExit> {
             return new Promise<ProcessExit>((resolve, reject) => {
                 let output = '';
                 proc.outputStream.on('data', chunk => output += chunk);
-                proc.onExit(async exit => {
+                proc.onClose(async exit => {
```

Comment:
@marechal-p, why was this required? Has the API change revealed a bug or why did you have to change `onExit` to `onClose`. Thanks!

Other than this, the API changes look good to me. if you feel like it, perhaps you can update the changelog.

Reply:
Hm, more like my eye caught this while I was editing the type above. Essentially here we try to handle the output of a process, but registering to `onExit` is bogus since it could fire before the streams are empty.

Reply:
OK, I do not mind if you add this change to the PR too, in fact, it's cool, but please split up into two commits and we can merge it. Thanks!